folly/io/async/AsyncSocket.cpp

```diff
@@ -662,7 +662,11 @@ void AsyncSocket::writeChain(WriteCallback* callback, unique_ptr<IOBuf>&& buf,
   constexpr size_t kSmallSizeMax = 64;
   size_t count = buf->countChainElements();
   if (count <= kSmallSizeMax) {
+    // suppress "warning: variable length array ‘vec’ is used [-Wvla]"
+#pragma GCC diagnostic push
```

Comment:
Other compilers don't know about this pragma. 

https://github.com/facebook/folly/blob/v2016.07.29.00/folly/Portability.h#L136-L159